floodwatch/src/js/components/Main.js

```diff
@@ -107,18 +107,20 @@ export class Main extends Component {
 
     const navs = this.state.user ? SIGNED_IN_NAVS : SIGNED_OUT_NAVS;
 
+    const navElem = (window.location.pathname !== "/generate") ? <Navigation navs={navs} /> : null;
```

Comment:
Ideally, we'd be entering the code from a different pathway, and wouldn't have to special case URLs. Do you think we could do this in the router in `App.js`? Then it'd be easier to debug.